app/components/edit/label_component.rb

```diff
@@ -3,16 +3,15 @@
 module Edit
   # Component for rendering a form label outside of a form field.
   class LabelComponent < ApplicationComponent
-    def initialize(label_text:, hidden_label: false, classes: [])
+    def initialize(label_text:, hidden_label: false, classes: [], required: false)
       @label_text = label_text
       @hidden_label = hidden_label
       @classes = classes
+      @required = required
       super()
     end
 
-    def call
-      tag.label(@label_text, class: classes)
-    end
```

Comment:
It looks like this component changes from an inline component to a template-y one. Can you say more about that change? I'm not sure if we've been 💯 consistent about this, but I do recall early on trying to use inline components if justifiable.

Reply:
Ah – this is leftover from when I needed to do more actual rendering (to render the `*`) and switched to a template, but is no longer necessary now that visual changes have been removed. I'll revert it.

Reply:
Removed.